var/spack/repos/builtin/packages/racon/package.py

```diff
@@ -25,10 +29,8 @@ class Racon(CMakePackage):
     conflicts('%gcc@:4.7')
     conflicts('%clang@:3.1')
 
+    patch('aarch64.patch', when='target=aarch64:')
```

Comment:
You could do something like:
```suggestion
    patch('https://path/to/aarch64.patch', sha256='...', when='target=aarch64:')
```
That way we don't need to store a 2,800 line patch file.

Reply:
This patch is come from another github repo:
https://github.com/DLTcollab/sse2neon/blob/master/sse2neon.h

I think it's not easy to get it?

Reply:
Hmm, you could use https://raw.githubusercontent.com/DLTcollab/sse2neon/master/sse2neon.h, but the checksum will change over time. If you can figure out a way to check out a specific commit, we can use that, otherwise we'll stick with the current approach.

Reply:
You can select a link to a specific commit of a file with

`https://github.com/DLTcollab/sse2neon/blob/65cdf9a/sse2neon.h`

(or replace `65cdf9a` with the commit you want)